apps/frontend-pwa/src/components/liveQuiz/QuestionArea.tsx

```diff
@@ -57,6 +59,14 @@ function QuestionArea({
 }: QuestionAreaProps): React.ReactElement {
   const t = useTranslations()
 
+  const auditLog = useAuditClient({
+    assessmentMode: process.env.NEXT_PUBLIC_IS_ASSESSMENT === 'true',
```

Comment:
right now these two are the same, as we have no audit log outside assessment mode